src/component/marker/markerHelper.ts

```diff
@@ -105,14 +105,14 @@ export function dataTransform(
 
     const data = seriesModel.getData();
     const coordSys = seriesModel.coordinateSystem;
-    const dims = coordSys.dimensions;
+    const dims = coordSys?.dimensions;
```

Comment:
Using the optional chain operator might bring some redundant code when compiling to a lower ES version that doesn't support it. For example, the above code may be compiled as,

```js
var dims = coordSys === null || coordSys === void 0 ? void 0 : coordSys.dimensions;
```

Compared to `&&`, it doesn't look neat.

Reply:
Thanks for letting me know about this!